### DISCUSSION TITLE
iPi Mocap Studio AI capture can't useable for AMD cards only works with Nvidia.

### DISCUSSION THREAD

**Discussion by KEDI103** [ROCm/ROCm#2645] [Ideas]
This feature missing from AMD which is unaccaptable for me. Without Nvidia I can't use AI capture to my paid sofware tracking.
https://docs.ipisoft.com/AI-based_Tracking
Heres also opened to reqest to AMD cards
https://forum.ipisoft.com/viewtopic.php?f=4&t=10599
I am gona build new pc soon and without this can't go with AMD it force me to switch Nvidia. It so importand program for me without it I can't capture still can use old method but why go old not good method instead of AI?

**Comment by fluidnumericsJoe** [comment]
Maybe take this issue up with the software providers and not ROCm..

**Reply by KEDI103** [reply]
Already
https://docs.ipisoft.com/AI-based_Tracking
but this is also base nvidia this direcly from nvidia is anything AMD ofer like this sdk dev?
[NVIDIA Maxine AR 0.8.2](https://github.com/NVIDIA/MAXINE-AR-SDK#readme)
